<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSVsmith</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="filename">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>550</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="credits">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Youssef Elebiary</string>
    </property>
   </widget>
   <widget class="QLabel" name="col_lbl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>550</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Colomn: </string>
    </property>
   </widget>
   <widget class="QLabel" name="row_lbl">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Row: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuTable">
    <property name="title">
     <string>Table</string>
    </property>
    <addaction name="actionAppend_Row"/>
    <addaction name="actionAppend_Column"/>
    <addaction name="actionInsert_Row"/>
    <addaction name="actionInsert_Column"/>
    <addaction name="actionRemove_Row"/>
    <addaction name="actionRemove_Column"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTable"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionGo_To">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go To</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAppend_Row">
   <property name="text">
    <string>Append Row</string>
   </property>
  </action>
  <action name="actionAppend_Column">
   <property name="text">
    <string>Append Column</string>
   </property>
  </action>
  <action name="actionInsert_Row">
   <property name="text">
    <string>Insert Row</string>
   </property>
  </action>
  <action name="actionInsert_Column">
   <property name="text">
    <string>Insert Column</string>
   </property>
  </action>
  <action name="actionRemove_Row">
   <property name="text">
    <string>Remove Row</string>
   </property>
  </action>
  <action name="actionRemove_Column">
   <property name="text">
    <string>Remove Column</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
